scripts/insert-energy-data.ts

```diff
@@ -0,0 +1,32 @@
+import { insertRawEnergyValues } from "@energyleaf/db/query";
+import fs from "node:fs";
+import path from "node:path";
+
+export async function insertEnergyData(args: string[]) {
+    const sensorId = args[0];
+    if (!sensorId) {
+        throw new Error("sensorId is required");
+    }
+
+    const filePath = path.join(process.cwd(), "apps", "web", "src", "lib", "demo", "demo.json");
+    const fileContent = fs.readFileSync(path.join(filePath, "demo.json"));
```

Comment:
Warum benutzt du hier nicht die async Variante? Ist zwar eigentlich nicht so wichtig, aber wäre meiner Meinung nach die schönere Variante

Reply:
Weil die Node async version, nicht mit promise funktioniert sondern über callbacks.
So ist es in dem Moment blocking aber denke das ist für den Zweck okay.